gef.py

```diff
@@ -229,12 +229,16 @@ def highlight_text(text: str) -> str:
     return "".join(ansiSplit)
 
 
-def gef_print(x: str = "", *args: Any, **kwargs: Any) -> Optional[int]:
-    """Wrapper around `print()`, using string buffering feature."""
-    x = highlight_text(x)
+def gef_print(*args: str, end="\n", sep=" ", **kwargs: Any):
+    """Wrapper around print(), using string buffering feature."""
     if gef.ui.stream_buffer and not is_debug():
-        return gef.ui.stream_buffer.write(x + kwargs.get("end", "\n"))
-    print(x, *args, **kwargs)
+        gef.ui.stream_buffer.write(
+            sep.join(highlight_text(a) for a in args) + end)
```

Comment:
`sep.join(highlight_text(a) for a in args)` is repeated, can be simplify imo

```python
    text = sep.join(highlight_text(a) for a in args)
    if gef.ui.stream_buffer and not is_debug():
        gef.ui.stream_buffer.write(text+ end)
        return

    print(text, end=end, **kwargs)
    return
```

Reply:
yeah that's good.
Note that you can click and drag the `+` for a comment over multiple lines, and that way you can give a multi line suggestion.

Reply:
how about this then?

```py
text = [highlight_text(a) for a in args]
if gef.ui.stream_buffer and not is_debug():
    gef.ui.stream_buffer.write(sep.join(text) + end)
else:
    print(text, sep=sep, end=end, **kwargs)
```

that way we are closer to how pythons builtin `print` works. (and IMO it would also be nice to start removing unnecessary trailing returns at the end of functions)

Reply:
yeah that's better actually, though we'd need to do `print(*text...` otherwise the print would but `[]` around it.

Reply:
(And hugsy likes those returns because he likes how they are explicit)